### PR TITLE
VACMS-15018: Fix PHPStan execution.

### LINKED ISSUE #15018: PHPStan tests failing to execute in GHA environment.
## Description

```
Note: Using configuration file /home/runner/work/va.gov-cms/va.gov-cms/phpstan.neon.
UnexpectedValueException thrown in /home/runner/work/va.gov-cms/va.gov-cms/docroot/core/tests/bootstrap.php on line 25 while loading bootstrap file /home/runner/work/va.gov-cms/va.gov-cms/phpstan_bootstrap.php: RecursiveDirectoryIterator::__construct(/home/runner/work/va.gov-cms/va.gov-cms/docroot/profiles): Failed to open directory: No such file or directory
```

## Acceptance Criteria

- [x] PHPStan tests execute successfully.

### PR DESCRIPTION
**Pull request by ndouglas**
Closes #15018.

### CODE REVIEW

File: docroot/modules/custom/va_gov_backend/src/Plugin/Validation/Constraint/PdfCheckValidator.php
Comment (ndouglas) [inline docroot/modules/custom/va_gov_backend/src/Plugin/Validation/Constraint/PdfCheckValidator.php:1]: This should register as an error.

**Comment by ndouglas** [comment]
Screenshot 2023-09-07 at 11 45 38 AM

File: docroot/profiles/.gitkeep
Comment (ndouglas) [inline docroot/profiles/.gitkeep:1]: Added because core PHPStan configuration assumes this directory exists, and will throw an error if it does not 😩

**Review by tonytaylor — DISMISSED** [review]
"My tea's gone cold, I'm wondering why
I got out of bed at all..."

Damn Stans and their presumptions - good deal otherwise.

**Pull request merged**